lib/node_modules/@stdlib/math/base/special/cpolar/lib/native.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var Float64Array = require( '@stdlib/array/float64' );
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Computes the absolute value and the phase of a double-precision complex floating-point number.
+*
+* @private
+* @param {Complex128} z - complex number
+* @returns {Array} absolute value and phase (in radians)
+*
+* @example
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var cpolar = require( '@stdlib/math/base/special/cpolar' );
```

Comment:
```suggestion
```

The require statement is not necessary in this example. We want to test the add-on interface, not the main export.